CI_DailyBuildUpdates.py

```diff
@@ -13,15 +13,19 @@ def main():
     parser = argparse.ArgumentParser(description='Get the daily buid updates')
     parser.add_argument('--info_type', default='brief', choices=['brief','detailed'], help='specify the job info type (brief or detailed)')
     parser.add_argument('--zone', help='specify the lease/zone', type= lambda arg:arg.split(','))
-    parser.add_argument('--ci_arch', default='p', choices=['p','z'], help='Specify the CI architecture type (p or z), default is p')
+    parser.add_argument('--job_type', default='p', choices=['p','z','pa'], help='Specify the CI job type (p or z or pa), default is p')
     args = parser.parse_args()
-    if args.ci_arch == 'p':
-        config_file = 'p_config.json'
-    elif args.ci_arch == 'z':
-        config_file = 'z_config.json'
+    if args.job_type == 'p':
+        config_file = 'p_periodic.json'
+    elif args.job_type == 'z':
+        config_file = 'z_periodic.json'
+    elif args.job_type == 'pa':
+        config_file = 'p_auxi.json'
     else:
-        print("Invalid argument. Please use p or z")
+        print("Invalid argument. Please use p or z or pa")
```

Comment:
Can we display the --job_type's help as well in this error? There should be some way to display it

Reply:
Thanks @Shilpa-Gokul  for observing this, actually this is a extra line of unwanted code, if a user provides any other value for job_type other than the one's mentioned in choices block, the argument parser will throw the error and display the help message, so I have removed that extra line of code.

```
D:\main-ci-monitoring\ci-monitoring-automation>python3 CI_JobHistory.py --job_type ap
usage: CI_JobHistory.py [-h] [--zone ZONE] [--job_type {p,z,pa}]
CI_JobHistory.py: error: argument --job_type: invalid choice: 'ap' (choose from 'p', 'z', 'pa')
```